lib/actions/addLinkToProjectAction.test.js

```diff
@@ -0,0 +1,20 @@
+const { expect } = require('chai');
+const { AddLinkToProjectAction } = require('./addLinkToProjectAction');
+
+describe('Add Action', ()=> {
+    xit('@asyncResolves : should add link to DB', async () => {});
+
+    it('@asyncResolves : should return an add object containing a response', async () => {
+        const addLinkToProjectAction = new AddLinkToProjectAction();
+
+        const response = await addLinkToProjectAction.execute(['google', 'google.com'], {links: {}})
+
+        const result = { label: 'google', link: 'google.com', project: { links: {} } }
```

Comment:
`result` caught me here. I was trying to figure out why the result was hard coded, only to realize this is the "expected" result. Maybe rename this for that reason.

Similarly, by setting an expectation about the `project: { links }` nodes, you're building a contract. Do we know we need these? Why? Do we care from the perspective of executing this specific action?

Reply:
Originally I thought that you didn't want me to explicitly type `project: {links}` but know I'm understanding that the entire expect line may be unnecessary.. 

What would we use to compare if we don't use expect?